book/chapter-02.md

```diff
@@ -1,16 +1,29 @@
 # Chapter 2: Client-Side Validation, Networks & Double Spends
+What is client side validation, and why is it so important? Well, currently all blockchain consensus protocols (e.g. Bitcoin, Ethereum, Ripple, etc.) all have a challenging limitation. And that is the fact that every fully participating node in the network must process every transaction. This is client side validation. It’s the validation of inputs (blocks) to the blockchain from the nodes overseeing the network. 
 
-**What is it?**
```

Comment:
please disregard all red highlights

Reply:
My writing only goes up to the "Privacy" bullet point relating to client side validation on the outline. Please feel free to critique - I added an interesting analogy/philosophy relating to client side validation and ASICS, but perhaps this is not the place in the book to be talking about it. Might be better suited for the "future" section?